src/instruments/src/ISIS/RollScale.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export const RollScale: React.FC = React.memo(() => {
+    const markerAngles = [-60, -45, -30, -20, -10, 10, 20, 30, 45, 60];
+
+    const radiusForMarkers = 138;
+
+    const tickLength = 20;
+    const dx = radiusForMarkers * Math.sin(30 * Math.PI / 180);
+    const dy = radiusForMarkers * Math.cos(30 * Math.PI / 180);
+
+    const createTick = (markerAngle: number) => {
```

Comment:
**suggestion:** Extract this into a `RollScale` component.

Reply:
I assume you mean a `RollScaleTick` component

Reply:
yes